flink-ai-flow/examples/workflow_on_event/README.md

```diff
@@ -0,0 +1,158 @@
+# Guidance on Running Workflow on Event Example
+
+The workflow_on_event project contains three workflows to demonstrate how the workflow execution on events can be useful
```

Comment:
how the `workflow_execution_on_events()` API works

Reply:
fixed in a19b1cc

Reply:
fixed in a19b1cc